README.md

```diff
@@ -769,6 +770,37 @@ Example invocation: `$ m . /lintPython/dirOfModules/makes/main`
 
 Example invocation: `$ m . /lintPython/module/cliMain`
 
+### lintWithLizard
+
+Using [Lizard](https://github.com/terryyin/lizard)
```

Comment:
Instead of placing links directly we use markdown references, checkout other links and you'll see how it works